woocommerce-services.php

```diff
@@ -265,9 +265,9 @@ public static function plugin_deactivation() {
 			 */
 			require_once __DIR__ . '/classes/class-wc-connect-wcst-to-wcshipping-migration-state-enum.php';
 
-			$migration_state = WC_Connect_Options::get_option( 'wcshipping_migration_state' );
+			$migration_state = get_option( 'wcshipping_migration_state' );
 			if ( $migration_state !== WC_Connect_WCST_To_WCShipping_Migration_State_Enum::COMPLETED ) {
-				WC_Connect_Options::update_option( 'wcshipping_migration_state', WC_Connect_WCST_To_WCShipping_Migration_State_Enum::COMPLETED );
+				update_option( 'wcshipping_migration_state', WC_Connect_WCST_To_WCShipping_Migration_State_Enum::COMPLETED );
```

Comment:
The `get_option()` doesn't type cast and `"12 !=== 12`. After the migration is done, the banner will continue to show. 
![image](https://github.com/Automattic/woocommerce-services/assets/572862/86818031-f60f-42b3-9c4b-75de0d01c8b9)